src/ddprof_cmdline.cc

```diff
@@ -78,6 +78,11 @@ bool watcher_from_event(const char *str, PerfWatcher *watcher) {
     watcher->tracepoint_name = watcher->desc;
     watcher->tracepoint_group = event_groupname;
   }
+
+  // Certain watcher configs get additional event information
```

Comment:
@r1viollet You had inlined this in the worker code in your PR.  I moved it here so it's more of an upfront configuration change.  Curious to learn whether this feels like a better change to you.